src/renderer/containers/App/handleUploadJobUpdates.ts

```diff
@@ -100,9 +101,7 @@ export function handleUploadJobUpdates(job: JSSJob, dispatch: Dispatch<any>) {
       dispatch(receiveFSSJobCompletionUpdate(fssJob));
     } else {
         // Otherwise, report progress
-        // TODO Multifile FSS jobs are currently identifiable by their service_fields.subfiles property,
-        //       but more ideally would just be labeled with something like "multifile: true".
-        if (fssJob.serviceFields?.subfiles) {
+        if (fssJob.serviceFields?.multifile) {
```

Comment:
Since this field ended up being `multifile` instead of `isMultifile`, maybe we should explicitly check this as `fssJob.serviceFields?.multifile === true`?